tests/performance/datasets/numerical.py

```diff
@@ -1,10 +1,20 @@
+"""Dataset Generators for numerical transformers."""
+
 import numpy as np
 
 from tests.performance.datasets.base import BaseDatasetGenerator
 
 
-class RandomNumericalGenerator(BaseDatasetGenerator):
-    """Generator that creates dataset of random integers."""
+def _add_nulls(array):
```

Comment:
Maybe we could move this to `base.py`, since other generators could use it too

Reply:
Moved to a new utils module and renamed it to `add_nans` instead.